pkg/collector/corechecks/ebpf/probe/ebpfcheck/probe.go

```diff
@@ -19,6 +19,7 @@ import (
 	"time"
 	"unsafe"
 
+	ebpf2 "github.com/DataDog/datadog-agent/pkg/util/ebpf"
```

Comment:
What do you think of `ebpfutils` or something similar ?
`ebpf` vs `ebpf2` is not super clear 😄